bucket/vnote.json

```diff
@@ -21,6 +21,9 @@
             "VNote"
         ]
     ],
+    "checkver": {
+        "github": "https://github.com/vnotex/vnote"
+    },
     "autoupdate": {
         "architecture": {
             "64bit": {
```

Comment:
Why do you propose random change, which cannot work?? There are no portable binaries. That is the reason why the autoupdate was removed...

![image](https://user-images.githubusercontent.com/13260377/128568412-07805001-f0fb-4fb0-ae38-faf774abb8c3.png)